src/urls.py

```diff
@@ -50,6 +51,7 @@
         # Django Admin, use {% url 'admin:index' %}
         path(settings.ADMIN_URL, admin.site.urls),
         path(f"{settings.ADMIN_URL}doc/", include("django.contrib.admindocs.urls")),
+        path("account/", AccountView.as_view(), name="account_page"),
```

Comment:
It is a bit weird (not sure myself what best) to have account/ and accounts/

Reply:
I want to make absolutely sure that I don't collide with django_allauth's url namespace without having to read docs carefully, so yeah, that's why they're separate. :)

Reply:
I would say then having "user_settings" or something like this is better.